From 3d7e2f140d4eb1dc7485ae867b786c13a3bc4b57 Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Sun, 19 May 2024 12:27:35 +0300
Subject: [PATCH] [SW-15774] board: adjust pldm scripts

New pldm scripts were added lately, and needs to be adjusted,
as part of the preparation for the product build rules split.

Change-Id: Ic659ee348d4271e2f58dacc561a5e696535d84f3
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/board/habanalabs/gaudi3/create-pldm-fwup-package-erom.sh b/board/habanalabs/gaudi3/create-pldm-fwup-package-erom.sh
index cddad6c..031d927 100755
--- a/board/habanalabs/gaudi3/create-pldm-fwup-package-erom.sh
+++ b/board/habanalabs/gaudi3/create-pldm-fwup-package-erom.sh
@@ -2,7 +2,7 @@
 
 source ${BUILDROOT_ROOT}/board/habanalabs/${DEV_PREFIX}/create-pldm-fwup-package.sh
 
-VERSION_FILE="${BUILDROOT_ROOT}/output/${DEV_PREFIX}_arc/build/zephyr-custom/build-agent-fw-loader/localversion-habanalabs"
+VERSION_FILE="${BUILDROOT_ROOT}/output/${DEV_PREFIX}_arc/build/agent-fwl-custom/localversion-habanalabs"
 EROM_VERSION_FILE="${BUILDROOT_ROOT}/output/${DEV_PREFIX}_arc/build/baremetal-custom/localversion-habanalabs"
 
 if [[ ! -f ${VERSION_FILE} ]]; then
diff --git a/board/habanalabs/gaudi3/create-pldm-fwup-package-margin-tool.sh b/board/habanalabs/gaudi3/create-pldm-fwup-package-margin-tool.sh
index bb2d85a..7f52307 100755
--- a/board/habanalabs/gaudi3/create-pldm-fwup-package-margin-tool.sh
+++ b/board/habanalabs/gaudi3/create-pldm-fwup-package-margin-tool.sh
@@ -5,7 +5,7 @@
 MARGIN_TOOL="${1}"
 COMPONENT_IDENTIFIER=$(($2))
 
-VERSION_FILE="${BUILDROOT_ROOT}/output/${DEV_PREFIX}_arc/build/zephyr-custom/build-agent-${MARGIN_TOOL}/localversion-habanalabs"
+VERSION_FILE="${BUILDROOT_ROOT}/output/${DEV_PREFIX}_arc/build/agent-${MARGIN_TOOL}-custom/localversion-habanalabs"
 
 if [[ ! -f ${VERSION_FILE} ]]; then
         echo "$0: ${MARGIN_TOOL} agent not built - cannot create PLDM package"
diff --git a/board/habanalabs/gaudi3/create-pldm-fwup-package-spi-flash.sh b/board/habanalabs/gaudi3/create-pldm-fwup-package-spi-flash.sh
index 6df42b0..7a0b51c 100755
--- a/board/habanalabs/gaudi3/create-pldm-fwup-package-spi-flash.sh
+++ b/board/habanalabs/gaudi3/create-pldm-fwup-package-spi-flash.sh
@@ -2,7 +2,7 @@
 
 source ${BUILDROOT_ROOT}/board/habanalabs/${DEV_PREFIX}/create-pldm-fwup-package.sh
 
-VERSION_FILE="${BUILDROOT_ROOT}/output/${DEV_PREFIX}_arc/build/zephyr-custom/build-agent-fw-loader/localversion-habanalabs"
+VERSION_FILE="${BUILDROOT_ROOT}/output/${DEV_PREFIX}_arc/build/agent-fwl-custom/localversion-habanalabs"
 
 if [[ ! -f ${VERSION_FILE} ]]; then
         echo "$0: FW loader agent not built - cannot create PLDM package"
diff --git a/board/habanalabs/gaudi3/create-pldm-fwup-package.sh b/board/habanalabs/gaudi3/create-pldm-fwup-package.sh
index 269f584..35277ff 100755
--- a/board/habanalabs/gaudi3/create-pldm-fwup-package.sh
+++ b/board/habanalabs/gaudi3/create-pldm-fwup-package.sh
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 
+source ${BUILDROOT_ROOT}/board/habanalabs/common/image-utility.sh
 PLDM_SCRIPTS_DIR="${BUILDROOT_ROOT}/scripts/pldm"
 PLDM_FWUP_JSON_CTOR_SCRIPT="${PLDM_SCRIPTS_DIR}/hl_pldm_fwup_package_json_creator.py"
 PLDM_FWUP_PACKAGE_CTOR_SCRIPT="${PLDM_SCRIPTS_DIR}/pldm_fwup_pkg_creator.py"
